src/AddImageChecks.cpp

```diff
@@ -20,28 +20,59 @@ class FindBuffers : public IRGraphVisitor {
         Buffer<> image;
         Parameter param;
         Type type;
-        int dimensions;
-        Result() : dimensions(0) {}
+        int dimensions {0};
```

Comment:
~everywhere else we do this we omit the space, as we do for a ctor:

    int dimensions{0};